components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -99,7 +118,24 @@ class ExperimentsTest {
 
     @Test
     fun `initialize experiments before glean`() {
-        resetExperiments()
+        // Because we interact with experimentsUpdater in resetExperiments in order to setup mocking
+        // of the update process, we need to use a scoped down version of the reset here to pass the
+        // verify zero interactions assertion
```

Comment:
Can't we use `verifyNoMoreInteractions()` here instead?

Reply:
No, it turns out I couldn't get it to work with `verifyNoMoreInteractions` as there the interaction appeared to happening in a way to make this fail too.  I can test for the 1 interaction in resetExperiments that I know about, so I changed it to check for that and no more interactions.